homeassistant/components/sonos/media_player.py

```diff
@@ -488,10 +487,13 @@ def update(self):
                 self._media_title = None
                 self._source_name = None
         elif available and not self._receives_events:
-            self.update_groups()
-            self.update_volume()
-            if self.is_coordinator:
-                self.update_media()
+            try:
+                self.update_groups()
+                self.update_volume()
+                if self.is_coordinator:
+                    self.update_media()
+            except SoCoException:
+                pass
```

Comment:
Why not log it failed to update something?

Reply:
The state goes to unavailable, I think that is enough.